Sources/AppcuesKit/AppcuesKit.docc/2.0MigrationGuide.md

```diff
@@ -0,0 +1,54 @@
+# 1.x to 2.0 Migration Guide
+
+## Overview
+
+There are a number of breaking changes from 1.x to 2.0, mostly related to the experience trait system. If you haven't implemented any custom actions or traits, the migration should be quite straightforward.
```

Comment:
> the migration should be quite straightforward
" the migration will involve little or no changes in your code." ?

Just thinking of something as inviting as possible to appease customers and support interactions and ease any fears there. It really should be generally no change at all for most customer usage it seems, agree?

Reply:
Yep, that's well phrased—I'll use that.